guidelines.json

```diff
@@ -8243,10 +8241,11 @@
 					"effort": "Medium",
 					"benefits": [
 						{
-							"Environment": "A strict sustainability or performance budget will reduce the chance of your website getting too large (and causing pollution transfers), which in turn will ensure it has a minimal impact on a visitor's device. This will have a direct impact on emissions reduction by forcing businesses to choose where to make reductions and efficiency savings.",
-							"Performance": "Keeping realistic goals regarding delivery size will push developers to optimize resource-heavy projects and reconsider using large tooling in place of lightweight alternatives. Moreover, A lower set target budget for a product or service will decrease the amount of time a browser spends transferring and rendering data.",
-							"Economic": "Customers will not have to keep upgrading devices to match the needs of a website that grows (unchecked) over time.",
-							"Conversion": "Reduced churn and page abandonment will occur due to the application or website becoming more performant. Potentially, your search engine ranking may perform better due to performance being a key indicator in PageRank algorithms."
+							"Conversion": "Improving performance will reduce churn and page abandonment. A website may also rank better on search engines thanks to performance being a key indicator in ranking algorithms.",
+							"Economic": "Reducing resource requirements means visitors will not have to keep upgrading devices to match the needs of digital products and services that are otherwise growing unchecked over time.",
+							"Environment": "Setting a strict sustainability or performance budget will reduce the chance of a digital product or service getting too large or resulting in pollution transfers, which will also ensure it has a minimal impact on a visitor's device. This has a direct impact on emissions by forcing businesses to choose where to make reductions and efficiency savings.",
```

Comment:
since we mention not upgrading devices in econ, we should mention it in environment too